djedi/admin/api.py

```diff
@@ -215,7 +220,12 @@ def post(self, request, uri):
             return self.render_plugin(request, context)
 
     def render_plugin(self, request, context):
+        ext = context['uri'].query['plugin'][0] \
+            if context['uri'].query and 'plugin' in context['uri'].query \
+            and context['uri'].query['plugin'] \
+            else context['uri'].ext
+        context['uri'] = mark_safe(context['uri'])
         return TemplateResponse(request, [
-            'djedi/plugins/%s/editor.html' % context['uri'].ext,
+            'djedi/plugins/%s/editor.html' % ext,
```

Comment:
This could be reverted right? List plugin renders with correct extension if i remember correctly?

Reply:
No because the URI of the sub-nodes will have the "list" extension still when requesting the editor. It was only the render-call where we simply changed which extension was used.

Reply:
Though I would like to find a nicer solution to this